src/providers/pronouns/Pronouns.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "providers/pronouns/alejo/PronounsAlejoApi.hpp"
+#include "providers/pronouns/UserPronouns.hpp"
+
+#include <shared_mutex>
+#include <optional>
+#include <unordered_map>
+
+namespace chatterino::pronouns {
+
+class Pronouns
+{
+private:
+    // mutex for editing the saved map.
+    std::shared_mutex mutex;
+    // Login name -> Pronouns
+    std::unordered_map<QString, UserPronouns> saved;
+    AlejoApi alejoApi;
```

Comment:
This should be moved below the `public` items (so that public items come first).